docusaurus/docs-internals/editor-controls/data-controls/transform.md

```diff
@@ -0,0 +1,101 @@
+| `id`                      | `string`                                                                                                                                                                        |    -    | The identifier of the key where the inputText will save your data                                                                                                                                                                 |
+| `type`                    | `string`                                                                                                                                                                        |    -    | Type should be `"inputText"` to use this control                                                                                                                                                                                  |
+| `label?`                  | `string`                                                                                                                                                                        |    -    | The label displayed on the left side of the control                                                                                                                                                                               |
+| `position?`               | `number`                                                                                                                                                                        |    -    | The position of the control in toolbar                                                                                                                                                                                            |
+| `devices?`                | `"all"` \| `"desktop"` \| `"responsive"`                                                                                                                                        | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `config?.disabled`        | `boolean`                                                                                                                                                                       | `false` | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
```

Comment:
about `config.disabled`

this is another props, not `disabled` from every option

`config.disabled` from your control is an array